assets/models/system/bytetrack_yolox_x_crowdhuman_mot17-private-half/spec.yaml

```diff
@@ -3,19 +3,56 @@ name: bytetrack_yolox_x_crowdhuman_mot17-private-half
 path: ./
 properties:
   finetune-min-sku-spec: 4|1|28|176
-  finetune-recommended-sku: Standard_NC6s_v3
+  finetune-recommended-sku: Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3,
+    Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3,
+    Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4,
+    Standard_ND96asr_v4, Standard_ND96amsr_A100_v4, Standard_ND40rs_v2
   finetuning-tasks: video-multi-object-tracking
-  inference-min-sku-spec: 2|1|14|28
-  inference-recommended-sku: Standard_NC6s_v3, Standard_NC12s_v3, Standard_NC24s_v3,
+  inference-min-sku-spec: 4|1|28|176
+  inference-recommended-sku: Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC12s_v3, Standard_NC24s_v3,
     Standard_NC24rs_v3, Standard_NC16as_T4_v3, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4,
-    Standard_NC4as_T4_v3, Standard_NC64as_T4_v3, Standard_NC8as_T4_v3, Standard_NC96ads_A100_v4,
+    Standard_NC64as_T4_v3, Standard_NC8as_T4_v3, Standard_NC96ads_A100_v4,
     Standard_ND40rs_v2, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4
-  model_id: bytetrack_yolox_x_crowdhuman_mot17-private-half
 tags:
   Preview: ''
   license: apache-2.0
   model_specific_defaults:
     apply_deepspeed: 'false'
     apply_ort: 'false'
   task: multi-object-tracking
-version: 3
\ No newline at end of file
+  openmmlab_model_id: bytetrack_yolox_x_crowdhuman_mot17-private-half
+  SharedComputeCapacityEnabled: ''
+  finetune_compute_allow_list:
+    [
+      Standard_NC4as_T4_v3,
+      Standard_NC6s_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC12s_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC24s_v3,
+      Standard_NC64as_T4_v3,
+      Standard_NC24ads_A100_v4,
+      Standard_NC48ads_A100_v4,
+      Standard_NC96ads_A100_v4,
+      Standard_ND96asr_v4,
+      Standard_ND96amsr_A100_v4,
+      Standard_ND40rs_v2
+    ]
+  inference_compute_allow_list:
+    [
+      Standard_NC4as_T4_v3,
+      Standard_NC6s_v3,
+      Standard_NC12s_v3,
+      Standard_NC24s_v3,
+      Standard_NC24rs_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC24ads_A100_v4,
+      Standard_NC48ads_A100_v4,
+      Standard_NC64as_T4_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC96ads_A100_v4,
+      Standard_ND40rs_v2,
+      Standard_ND96amsr_A100_v4,
+      Standard_ND96asr_v4
+    ]
+version: 4
```

Comment:
if version was incremented before and model is not yet published, then we can revert back version change

Reply:
checked, its 3 in prod